capellambse_context_diagrams/context.py

```diff
@@ -409,6 +438,24 @@ def _create_diagram(self, params: dict[str, t.Any]) -> cdiagram.Diagram:
         return class_diagram
 
 
+def add_context(data: _elkjs.ELKOutputData) -> None:
+    ids: set[str] = set()
+
+    def get_ids(obj: _elkjs.ELKOutputNode) -> None:
+        if obj["id"] and not obj["id"].startswith("g_"):
+            ids.add(obj["id"])
+        for cobj in obj.get("children", []):
+            if cobj["type"] == "node":
+                get_ids(cobj)
+
+    for child in data["children"]:
+        if child["type"] == "node":
+            get_ids(child)
+
+    for child in data["children"]:
+        child["context"] = list(ids)  # type: ignore[typeddict-unknown-key]
```

Comment:
This is a very simple algorithm calculating a single context/set of IDs. Currently all context diagram types are just one connected graph. So algorithmically this is fine. We still need a doc-string here.